### ISSUE TITLE
Registration problem

### ISSUE THREAD

**Issue by Billy-102** [WongSaang/chatgpt-ui#232]
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/312ee098-bcf3-457b-ab16-3cbe1460e535)
After deploying through docker compose, I start test the client by register a new account. However, here comes the problem (400 bad request), is it due to my wrong configuration?

**Comment by Suleman-Elahi** [comment]
show the compos file you are using and the platform you are deploying it .....

**Comment by Billy-102** [comment]
I just modified the SERVER_DOMAIN and APP_DOMAIN:
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/22becf09-6384-4cd7-9811-5ebcc817defc)
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/88c35042-452f-4f9c-bf7e-2c024eadd45f)

**Comment by Suleman-Elahi** [comment]
If you are using Google Cloud or AWS then have you allowed those ports through the firewall?

**Comment by ilovefreesw** [comment]
Try using the default SQLite database. You are using Postgres.

**Comment by Billy-102** [comment]
I have left the postgres configuration commented to take the SQLite as default. Should I configure it explicitly?
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/491715fb-7a35-40a7-ba93-2393650bc23e)

**Comment by Suleman-Elahi** [comment]
Is it working now ? 

The problem with your setup is that you are using the localhost. That will not work.

Set DB_URL to what @ilovefreesw mentioned above.

**Comment by Billy-102** [comment]
I set this but still got the error:
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/7a1105f7-57ce-4c32-b5ae-6d668ab318c2)

![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/622ffcf0-268e-4654-aa0f-eec5472a05b9)

**Comment by Suleman-Elahi** [comment]
That's strange .. post the log of `docker-compose up` command.

**Comment by Billy-102** [comment]
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/eeeb0c03-68a0-4edb-9b91-e09e582ac509)
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/0d4cf727-1f64-40dd-aa24-8b510904d7ac)
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/9d48b76a-2988-49af-a821-52bc2252a211)
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/54e65d10-a8e6-4e5f-ac34-44affdf9f266)
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/0bcf726b-87e1-4d76-8806-130c6784d4f9)
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/ef6bbee2-9178-411b-af56-0d29cc03f5ad)

**Comment by Suleman-Elahi** [comment]
Yes, it is the problem with the Postgres srever name. I think you are using older version of Docker. Whatever the OS is this, find its IP address.

Tell me the output of `ip addr show` command.

**Comment by Billy-102** [comment]
![image](https://github.com/WongSaang/chatgpt-ui/assets/94583069/d1f1add8-cc3e-46af-a0f0-ebe59239ff13)

**Comment by Suleman-Elahi** [comment]
Then use this as the `DB_URL = postgres://postgres:postgrespw@172.31.88.184:49153/chatgpt`

And if it still doesn't work then create a new EC2 instance with Ubuntu 22.04 or Ubuntu 20.04 image.

**Comment by Billy-102** [comment]
I tried this, but the problem remains unresolved.

**Comment by Suleman-Elahi** [comment]
Did it work when you used the default SQLite database?

**Comment by Billy-102** [comment]
As for creating a new EC2 instance, I have not tried. Could you please give me some more detailed instructions? Thank you.

**Comment by Suleman-Elahi** [comment]
Where are you using it right now ? on Which cloud provider and operating system?

**Comment by Suleman-Elahi** [comment]
Follow this tutorial: I think this is the same. https://www.youtube.com/watch?v=VKDU10gbYVQ

**Comment by Billy-102** [comment]
AWS,  Ubuntu-22.04.1

**Comment by Billy-102** [comment]
> Follow this tutorial: I think this is the same. https://www.youtube.com/watch?v=VKDU10gbYVQ
> 
> But it uses DigitalOcean ..so chnage that for yourself..of if you can afford DO then follow this tutorial.

Thank you, I'll take it as a reference.
